### DISCUSSION TITLE
PPOCRLabel.py：ModuleNotFoundError: No module named 'win32com'

### DISCUSSION THREAD

**Discussion by aqingGitHub** [PaddlePaddle/PaddleOCR#13394] [Q&A]
在使用PPOCRLabel打标时发现一些无用的图片，单击界面右下角的“删除图像”后出现如下错误，结果无用的图像没法被删除。
unicodeFilePath is D:\ICP Video\C0441\223.jpg
QPixmap::scaled: Pixmap is a null pixmap
Traceback (most recent call last):
  File "PPOCRLabel.py", line 1960, in deleteImg
    from win32com.shell import shell, shellcon
ModuleNotFoundError: No module named 'win32com'

**Comment by GreatV** [comment]
win32com是Python的一个模块，部分PyWin32扩展包，用于访问Windows的COM（组件对象模型）接口。你可以使用它来与Windows应用程序（例如Excel、Word）进行交互。要安装PyWin32，可以使用以下命令：

```bash
pip install pywin32
```

它提供了许多Windows API的访问接口，适用于各种Windows编程任务。
